gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.0-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.4-bin.zip
```

Comment:
```suggestion
distributionUrl=https\://services.gradle.org/distributions/gradle-7.4-all.zip
```

Reply:
4da91cf4403539b9ce742d6f706c07dd54f6c452

And limit wrapper in fde03d3c067a30446f5b290699c6cb17afc1efbe